clients/drcachesim/tracer/physaddr.cpp

```diff
@@ -93,8 +98,12 @@ physaddr_t::init()
 #ifdef LINUX
     // Some threads may not do much, so start out small.
     constexpr int V2P_INITIAL_BITS = 9;
-    hashtable_init_ex(&v2p_, V2P_INITIAL_BITS, HASH_INTPTR, /*strdup=*/false,
-                      /*synch=*/false, nullptr, nullptr, nullptr);
+    // The hashtable lookup performance is important.
+    // A non-open-address hashtable is about 3x slower due to the extra
```

Comment:
Curious about the terminology, why not just say closed address hashtable instead of non-open address hashtable? Maybe the latter includes some types of hashtable addressing schemes I'm not aware of?

Reply:
I think I just never hear anyone say "closed address" and so I don't use the term: in my head it's always been open address or "not".

Reply:
Changed it to say "closed-address".